packages/mermaid/src/diagrams/class/classDiagram.spec.ts

```diff
@@ -264,6 +264,86 @@ class C13["With Città foreign language"]
       const str = 'classDiagram\n' + 'note "test"\n';
       parser.parse(str);
     });
+
+    it('should handle note with "cssClass" in it', function () {
+      const str = 'classDiagram\n' + 'note "cssClass"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "callback" in it', function () {
+      const str = 'classDiagram\n' + 'note "callback"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "link" in it', function () {
+      const str = 'classDiagram\n' + 'note "link"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "click" in it', function () {
+      const str = 'classDiagram\n' + 'note "click"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "note" in it', function () {
+      const str = 'classDiagram\n' + 'note "note"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "note for" in it', function () {
+      const str = 'classDiagram\n' + 'note "note for"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "<<" in it', function () {
+      const str = 'classDiagram\n' + 'note "<<"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with ">>" in it', function () {
+      const str = 'classDiagram\n' + 'note ">>"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "href " in it', function () {
+      const str = 'classDiagram\n' + 'note "href "\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "call " in it', function () {
+      const str = 'classDiagram\n' + 'note "call "\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "~" in it', function () {
+      const str = 'classDiagram\n' + 'note "~"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "``" in it', function () {
+      const str = 'classDiagram\n' + 'note "``"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "_self" in it', function () {
+      const str = 'classDiagram\n' + 'note "_self"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "_blank" in it', function () {
+      const str = 'classDiagram\n' + 'note "_blank"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "_parent" in it', function () {
+      const str = 'classDiagram\n' + 'note "_parent"\n';
+      parser.parse(str);
+    });
+
+    it('should handle note with "_top" in it', function () {
+      const str = 'classDiagram\n' + 'note "_top"\n';
+      parser.parse(str);
+    });
```

Comment:
1. You could wrap all the examples in `describe` group for more informative tests output.
2. It is possible to cycle through all these examples with [it.each](https://vitest.dev/api/#test-each) or even with plain javascript

Reply:
Fair enough, did not know this. So, I'll give it a shot now.

Reply:
Thanks for all the feedback!

Reply:
These were wrapped in an `it.each` loop as you suggested.